scripts/globals/mobskills/throat_stab.lua

```diff
@@ -19,7 +19,7 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
-    if mob:getFamily() == 271 then -- Jailer of Love, uses only animation.
+    if mob:getPool() == 271 then -- Jailer of Love, uses only animation.
```

Comment:
This change needs to be removed. Probably need to remove this whole bit about checking for JoL since you made the separate throat stab 2hr lua.